docs/content/en/docs/install/upgrade.md

```diff
@@ -26,6 +26,78 @@ If you installed your Keptn instance from the Manifest,
 additional steps are required to use the Helm Chart to upgrade.
 Contact us on Slack for assistance.
 
+## Upgrade from installation via manifests using Helm
+
+Keptn v.0.7.0 and later can be installed with Helm charts;
+Keptn v.0.8.3 can only be installed with Helm.
+If you previously installed Keptn from manifests,
+you can not directly upgrade to with Helm but must back up your Keptn data,
+then reinstall Keptn from a Helm chart.
+
+To not loose all of your data, we encourage you to do a backup of the Keptn CRs,
+`Namespaces`, `Secrets` and `ConfigMaps`.
+Your applications (`Pods`, `Deployments`,
+`StatefulSets`, `DeamonSets`,...) won't be part of the backup and you should do
+it manually.
+
+To do this, copy the following code into `backup-script.sh` file:
+
+```shell
+#!/bin/bash
+
+kubectl get ns -oyaml >> backup.yaml
+kubectl get configmaps -A -oyaml >> backup.yaml
+kubectl get secrets -A -oyaml >> backup.yaml
+
+resources=$(kubectl get crd -n keptn-lifecycle-toolkit-system --no-headers -o custom-columns=NAME:.metadata.name)
+
+for name in "${resources[@]}"; do
+   kubectl get $name -A -oyaml >> backup.yaml
+done
+```
+
+Then make your file executable and execute the script:
+
+```shell
+chmod +x backup-script.sh && ./backup-script.sh
+```
+
+This creates a manifest called `backup.yaml` that includes all the CRs, `Namespaces`, `Secrets`
+and `ConfigMaps`.
+
+> **Note** This only backs up you Keptn configuration data.
+   Use standard Kuberetes tools and practices to back up your entire cluster.
+
+1. Completely remove your Keptn installation with the following command sequence:
+
+```shell
+your-keptn-version=<your-keptn-version>
+kubectl delete -f \
+     https://github.com/keptn/lifecycle-toolkit/releases/download/$your-keptn-version/manifest.yaml
+```
+
+1. Use Helm to install a clean version of Keptn:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+For information about  advanced installation options, refer to
+[Modify Helm configuration options](install.md).
+
+After the installation finishes, restore the Keptn resources from your backup:
+To apply them execute:
+
+```shell
+kubectl apply -f backup.yaml
+```
+
+> **Note** Please be aware that some Keptn applications may start the deployment from the
+beginning (if the `Deployments` are in place) and the system is not guaranteed to return
+to the exact state it was in before re-installation, even if you created the backup correctly.
```

Comment:
can you apply a CR with a Stuatus? Won't that field be lost?

Reply:
you can and yes it will be lost, but that is expected. The backup is not 100% and we declare that the whole process will start from scratch if the resources would be applied, we also declare in the docs that there is no guarantee. Users need to count with loose of data

Reply:
what's the purpose of the backup then?! 🤔

Reply:
docs adapted according to our private discussion to not create a backup of all Keptn resources but only the manifests which were applied to the cluster by user (or GitOps solution)